docs/git/index.md

```diff
@@ -0,0 +1,220 @@
+
+That's right, https://github.com/EagerLearner/TODO!
+
+This is called a _fork_. It's a copy of the repository, where you're allowed to make changes. So when Github Desktop just asked us "Do you want to fork this repository?", what it's really saying is "You're not allowed to make changes to CodeYourFuture's repository, would you like to make your own copy on Github where you _are_ allowed to make changes, and put your changes there?"
+
+That sounds like exactly what we want to do, so we'll click the "Fork This Repository" button.
+
+Then Github asks us whether we want to fork "To contribute to the parent project" (i.e. because we want to work with CodeYourFuture) or "For my own purposes" (i.e. because we want to do our own thing apart from CodeYourFuture). We want to work with CodeYourFuture, so we'll select "To contribute to the parent project" and press Continue.
+
+Now if we press "Push origin", it will copy our changes to our fork on Github.
+
+TODO: Insert graphic showing four repositories: Teacher's computer, CodeYourFuture, EagerLearner Fork, EagerLearner's computer.
+
+## Making a Pull Request
+
+Now that we've pushed our homework to our fork, we need to tell CodeYourFuture about it, so that the volunteers know to look at it!
+
+We do this with something called a _pull request_. This is a slightly weird name, in the context of homework. Normally, when people push changes to a fork on Github, they're doing so because they want the person who owns the repository to look at the changes, and pull the changes into their repository. For example, this webpage we're reading the syllabus on right now is hosted on Github, and if someone spots a typo, they can fix it, push it to their fork, and request for CodeYourFuture to pull their change into CodeYourFuture's version (hence the name a pull request - "requesting to pull in changes"). We call pulling someone's change into a repository "merging" the change (some people use the phrase "merge request" instead of "pull request").
+
+For submitting homework, you're going to create a pull request, and a volunteer will look at it and give you feedback, but we won't be pulling your homework into CodeYourFuture's copy (then the next student would have the answers when they tried to read the questions!). You'll be creating pull requests, but we won't actually merge your changes into the repository.
+
+In Github Desktop, if you open the Branch menu, and click "Create Pull Request", it will open your web browser at Github, and how you the changes you're about to make a pull request for. This is another great time to check that you're happy with your homework (if you're not, go back to VS Code, make your changes, commit them, push them to origin again, and refresh this page).
+
+If you're happy, press the "Create pull request" button. Fill in the details in the form, so that the volunteers know what they're meant to be reviewing, and press "Create pull request".
```

Comment:
I'd suggest that we try to standardise on some PR title/description naming. Something like "CITY CLASS_NO - FIRST_NAME LAST_NAME - WEEK_NO"? This would make it easier to distribute review among volunteers and know who you're marking.